app/index.html

```diff
@@ -43,6 +43,8 @@
 
     <link href='https://api.mapbox.com/mapbox.js/v2.3.0/mapbox.css' rel='stylesheet' />
     <link href='https://api.mapbox.com/mapbox-gl-js/v0.39.1/mapbox-gl.css' rel='stylesheet' />
+    <link rel="stylesheet" href="https://unpkg.com/react-select/dist/react-select.css">
```

Comment:
Is the CSS being packaged using the vendor CSS scripts, or inserted globally? I feel there is a redundancy here unless I'm not understanding how you imported the CSS.

Reply:
It's not packed. I just went with loading how it works for the mapbox vendor.

Reply:
Ok, then remove the react-select scss imports I guess?

Reply:
I can't figure out why it doesn't seem to get built for me. I've asked @ricardomestre for help. But I might merge this and patch that in later.